### PR TITLE
Issue/1750 aztec design changes

### LINKED ISSUE #1750: Aztec editor: Design changes
It looks like there is no option to customise the styles/colors for the Axztec Toolbar at the moment. This is needed by the Woo app when trying to update product description. Once Aztec is updated, we would need to incorporate changes to the Aztec editor to fix the design changes required.

- Change toolbar border color to `#E6E6E6` 
- Change toolbar background to white
- Change toolbar icon colors to `#595959`

LEFT - What is needed
RIGHT - Current implementation
 .

### PR DESCRIPTION
**Pull request by anitaa1990**
Fixes #1750 by adding design changes to the Aztec Toolbar editor. 

~**Note that this PR is in draft till this [Aztec PR](https://github.com/wordpress-mobile/AztecEditor-Android/pull/900) is merged and then we would need to update the hash to this branch before it can be merged.**~

I decided to implement this change in the dark mode branch since it updates styles and colors to the Aztec editor. 

#### Light Mode:
. . 

#### Dark mode
. . 

### To test
- Testing in API 28, 28, 21 and 23 with both dark and light mode would be great.
- General smoke testing the Aztec toolbar to ensure that everything is working as expected.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by AmandaRiu** [comment]
Nice! You figured out the toolbar styling! That's amazing. Now I'm wondering how I missed it. Let me know when this one is ready for an official review.

**Comment by anitaa1990** [comment]
Thanks @AmandaRiu! This PR is in draft till this [Aztec PR ](https://github.com/wordpress-mobile/AztecEditor-Android/pull/900) changes have been reviewed

**Comment by AmandaRiu** [comment]
> Thanks @AmandaRiu! This PR is in draft till this Aztec PR changes have been reviewed.

Oh! Ok now it all makes better sense. You've added those changes to the aztec library. Very cool. Nice work!

**Comment by anitaa1990** [comment]
@AmandaRiu PR ready for review 👍

**Comment by anitaa1990** [comment]
Thanks for the review @AmandaRiu! I'm good to make those changes to Aztec given that it's consistent with how our bottom navigation bar is displayed. I think it warrants a design review from @Garance91540 since this modifies the designs a little bit :D

File: WooCommerce/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -403,6 +403,12 @@ Use this file to override styles in the styles_base.xml file.
         <item name="codeBackgroundColor">@android:color/transparent</item>
         <item name="android:inputType">textNoSuggestions|textMultiLine</item>
     </style>
+    <style name="AztecToolbarStyle">
+        <item name="advanced">false</item>
+        <item name="toolbarIconNormalColor">@color/color_on_surface_medium</item>
+        <item name="toolbarIconHighlightColor">@color/color_primary</item>
+        <item name="toolbarIconDisabledColor">@color/color_on_surface_disabled</item>
+    </style>
```
[the remark this answers is not in the document]
Comment (anitaa1990) [reply WooCommerce/src/main/res/values/styles.xml]: Oops! My bad! I added everything to deprecated files 🤦‍♀️! Fixed in 7c91e3b

File: WooCommerce/src/main/res/layout/fragment_aztec_editor.xml [hunk 2]
```diff
@@ -11,6 +11,8 @@
         style="@style/Woo.AztecToolbar"
         android:layout_width="match_parent"
         android:layout_height="@dimen/min_tap_target"
+        app:toolbarBackgroundColor="@color/color_surface"
+        app:toolbarBorderColor="@color/wc_border_color"
```
[the remark this answers is not in the document]
Comment (anitaa1990) [reply WooCommerce/src/main/res/layout/fragment_aztec_editor.xml]: Fixed in 7c91e3b 👍

**Comment by anitaa1990** [comment]
Fixed the requested changes @AmandaRiu! PR ready for another round 👍

**Pull request merged**